react-ui/src/App.js

```diff
@@ -9,6 +9,9 @@ import {
   Route,
   Redirect
 } from "react-router-dom";
+import WorkerInputpage from './components/Layouts/InputWorkerTimes/WorkerInputpage'
+
```

Comment:
Hi Kyle,
Minor concern but is it possible to remove the unnecessary whitespaces on line 13, 14 and 15 so we can reduce the file sizes and increase file clarity?

Reply:
Deleted Whitespace as requested